sql/analyzer/load_show_triggers.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 Liquidata, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package analyzer
+
+import (
+	"github.com/dolthub/go-mysql-server/sql"
+	"github.com/dolthub/go-mysql-server/sql/parse"
+	"github.com/dolthub/go-mysql-server/sql/plan"
+)
+
+func loadShowTriggers(ctx *sql.Context, a *Analyzer, n sql.Node, scope *Scope) (sql.Node, error) {
+	span, _ := ctx.Span("loadShowTriggers")
+	defer span.Finish()
+
+	return plan.TransformUp(n, func(n sql.Node) (sql.Node, error) {
+		switch node := n.(type) {
+		case *plan.ShowTriggers:
```

Comment:
Probably want this as a top-level ok cast outside the transform. More efficient and documents the type of node the function runs on

Reply:
At first I did have it as a top-level ok cast, but changed it to this as `ShowTriggers` may be embedded in a `Filter` node. With that in mind, would you rather this approach or nested ok casts?

Reply:
That's a good point! Fine to have it as is.